jwt-authorizer/templates/jwt-authorizer-configmap.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jwt-authorizer-config
```

Comment:
Names should probably be prefixed by {{ .Release.Name }}

Reply:
I added the _helpers.tpl and used that